README.md

```diff
@@ -1,178 +1,253 @@
+- [Tutorials](https://docs.timescale.com/tutorials/latest/)
 
-### Installation
+## Create continuous aggregates
 
-Installation options are:
+Continuous aggregates are designed to make queries on very large datasets run faster. They continuously and incrementally refresh a query in the background, so that when you run such query, only the data that has changed needs to be computed, not the entire dataset. This is what makes them different from regular PostgreSQL [materialized views](https://www.postgresql.org/docs/current/rules-materializedviews.html), which cannot be incrementally materialized and have to be rebuilt from scratch every time you want to refresh it.
 
-- **[Timescale Cloud](https://tsdb.co/GitHubTimescale)**: A fully-managed TimescaleDB in the cloud, is
-  available via a free trial. Create a PostgreSQL database in the cloud with TimescaleDB pre-installed
-  so you can power your application with TimescaleDB without the management overhead.
+For example, create a continuous aggregate view for daily weather data in two simple steps:
 
-- **Platform packages**: TimescaleDB is also available pre-packaged for several platforms such as
-  Linux, Windows, MacOS, Docker, and Kubernetes. For more information, see [Install TimescaleDB](https://docs.timescale.com/self-hosted/latest/install/).
+1. Create a materialized view:
+
+   ```sql
+   CREATE MATERIALIZED VIEW conditions_summary_daily
+   WITH (timescaledb.continuous) AS
+   SELECT
+     device,
+     time_bucket(INTERVAL '1 day', time) AS bucket,
+     AVG(temperature),
+     MAX(temperature),
+     MIN(temperature)
+   FROM
+     conditions
+   GROUP BY
+     device,
+     bucket;
+   ```
 
-- **Build from source**: See [Building from source](https://github.com/timescale/timescaledb/blob/main/docs/BuildSource.md).
+1. Create a policy to refresh the view every hour:
 
-  We recommend not using TimescaleDB with PostgreSQL 17.1, 16.5, 15.9, 14.14, 13.17, 12.21.  
-  These minor versions [introduced a breaking binary interface change][postgres-breaking-change] that, 
-  once identified, was reverted in subsequent minor PostgreSQL versions 17.2, 16.6, 15.10, 14.15, 13.18, and 12.22.
-  When you build from source, best practice is to build with PostgreSQL 17.2, 16.6, etc and higher. 
-  Users of [Timescale Cloud](https://console.cloud.timescale.com/) and Platform packages built and 
-  distributed by Timescale are unaffected.
+   ```sql
+   SELECT
+     add_continuous_aggregate_policy(
+       'conditions_summary_daily',
+       start_offset => INTERVAL '1 month',
+       end_offset => INTERVAL '1 day',
+       schedule_interval => INTERVAL '1 hour'
+   );
+   ```
+See more:
 
-## Resources
+- [About continuous aggregates](https://docs.timescale.com/use-timescale/latest/continuous-aggregates/)
+- [API reference](https://docs.timescale.com/api/latest/continuous-aggregates/create_materialized_view/)
 
-### Architecture documents
+## Back up, replicate, and restore data
```

Comment:
I don't think this is needed in the README. This is kind of advanced and should be covered in docs.

Reply:
removed